bigbluebutton-html5/imports/ui/components/dropdown/list/item/component.jsx

```diff
@@ -2,26 +2,49 @@ import React, { Component, PropTypes } from 'react';
 import styles from '../styles';
 import _ from 'underscore';
 import cx from 'classnames';
-
 import Icon from '/imports/ui/components/icon/component';
+import { defineMessages, injectIntl } from 'react-intl';
+
+const intlMessages = defineMessages({
+  exitfullscreenLabel: {
```

Comment:
KDSBrowne pointed out that we have "Full screen" localized here https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/component.jsx#L24-L27